docs/source/ref/models/fields.rst

```diff
@@ -299,6 +299,17 @@ These indexes use 0-based indexing.
     As described above for :class:`EmbeddedModelField`,
     :djadmin:`makemigrations` does not yet detect changes to embedded models.
 
+``EncryptedCharField``
+----------------------
+
+.. class:: EncryptedCharField
+
+    Field that encrypts its value before saving to the database.
+
+.. admonition:: Migrations support is limited
+
+    :djadmin:`makemigrations` does not detect changes to encrypted fields.
```

Comment:
This wording was copied from changes to embedded fields, but the behavior here is different. makemigrations  will detect changes to encrypted fields but trying to run those migrations will fail with server errors since changes to encrypted fields aren't supported by MongoDB.